.github/workflows/daily-boot-iso.yml

```diff
@@ -0,0 +1,42 @@
+name: Build daily Rawhide+COPR boot.iso
+on:
+  schedule:
+    - cron: 0 22 * * *
+  # be able to start this action manually from a actions tab when needed
+  workflow_dispatch:
+
+jobs:
+  boot_iso:
+    name: Build boot.iso
+    runs-on: ubuntu-latest
+    steps:
+      # lorax and its dependencies do not exist in Ubuntu, and we want to test the latest Fedora version anyway
+      # lorax does mounts and uses loop devices, thus needs to be privileged
+      - name: Start fedora container
+        run: |
+          mkdir data
+          docker run --name fedora -d --privileged --network host -v $PWD/data:/data fedora:latest sleep infinity
```

Comment:
However, I wonder why ``sleep infinity``. I thought it should be fine to use ``docker create`` without a command, or am I missing something?

Reply:
`create` doesn't start the container, thus you can't run anything in it. The container will run as long as its pid 1 is running, and thus that needs to be something innocuous. `bash -i` is an ususal suspect, but that doesn't work with `--detach`, it would just immediately exit.

Reply:
I see. Thanks for explanation!